packages/connection-form/src/utils/set-app-name-if-missing.ts

```diff
@@ -14,7 +20,11 @@ export function setAppNameParamIfMissing(
     const searchParams =
       connectionStringUrl.typedSearchParams<MongoClientOptions>();
     if (!searchParams.has('appName') && defaultAppName !== undefined) {
-      searchParams.set('appName', defaultAppName);
+      const appName = `${defaultAppName}${
```

Comment:
I used `-` as a separator, open to suggestions if this can be problem causing in ways